core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -763,21 +760,19 @@ public synchronized void startInternal() throws IOException {
         )
         .collect(Collectors.toSet());
     mRaftGroup = RaftGroup.valueOf(RAFT_GROUP_ID, peers);
-    initServer();
-    super.registerMetrics();
```

Comment:
this line is removed?

Reply:
yes, `super.registerMetrics()` is also called at the end of `initServer`. Since `startInternal` always calls `initServer` I thought it was an unnecessary duplicate line and deleted it.